pkg/router/template/template_helper.go

```diff
@@ -321,7 +317,7 @@ func generateHAProxyMap(name string, td templateData) []string {
 
 // clipHAProxyTimeoutValue prevents the HAProxy config file
 // from using timeout values specified via the haproxy.router.openshift.io/timeout
-// annotation that exceed the maximum value allowed by HAProxy.
+// annotation that exceed the maximum value allowed by HAProxy, or by time.ParseDuration.
 // Return the parameter string instead of an err in the event that a
 // timeout string value is not parsable as a valid time duration.
```

Comment:
```suggestion
// Return the empty string instead of an err in the event that a
// timeout string value is not parsable as a valid time duration.
```